docker_hub/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:latest
+ARG ADAPT_VERSION
+RUN apk --update add --no-cache nodejs npm openssh \
```

Comment:
Adapt will also need git. And I think you can do just openssh-client instead of both client and server.

Reply:
Done.